src/hotspot/os/posix/signals_posix.cpp

```diff
@@ -594,6 +594,12 @@ int JVM_HANDLE_XXX_SIGNAL(int sig, siginfo_t* info,
   }
 #endif
 
+  if (!signal_was_handled && sig == BREAK_SIGNAL) {
+    assert(!ReduceSignalUsage, "Should not happen with -Xrs/-XX:-ReduceSignalUsage");
```

Comment:
Sorry, this is a typo. -Xrs=-XX:+ReduceSignalUsage. It should be + instead of - before ReduceSignalUsage. 
I will update in next revision.